app/views/directives/osc-persistent-volume-claim.html

```diff
@@ -97,19 +101,32 @@
     </div>
 
     <div class="form-group" >
-      <label class="required">Access Mode</label><br/>
-      <label class="radio-inline">
-        <input type="radio" name="accessModes" ng-model="claim.accessModes" value="ReadWriteOnce" aria-describedby="access-modes-help" ng-checked="true" >
-        Single User (RWO)
-      </label>
-      <label class="radio-inline">
-        <input type="radio" id="accessModes" name="accessModes" ng-model="claim.accessModes" value="ReadWriteMany" aria-describedby="access-modes-help" >
-        Shared Access (RWX)
-      </label>
-      <label class="radio-inline">
-        <input type="radio" name="accessModes" ng-model="claim.accessModes" value="ReadOnlyMany" aria-describedby="access-modes-help">
-        Read Only (ROX)
+      <label class="required">
+        Access Mode
+        <span ng-if="claim.storageClass && (claim.storageClass | storageClassAccessMode)">
+          <small>(cannot be changed)</small>
```

Comment:
Missed this as well, but the "cannot be changed" font style here is different here compare to size. It would be good to be consistent.

![openshift web console 2018-02-26 12-48-09](https://user-images.githubusercontent.com/1167259/36686131-63f667d2-1af3-11e8-80a1-e303c12395ef.png)

Reply:
@spadgett - Remove code and here is the new screen shot:
![screenshot from 2018-02-26 11-35-47](https://user-images.githubusercontent.com/18728857/36688621-431ad128-1aea-11e8-89b5-a57cb52a7db0.png)